unipipe/blueprints/unipipe-osb-aci.tf.js

```diff
@@ -99,9 +112,9 @@ resource "azurerm_container_group" "unipipe_osb_with_ssl" {
     }
 
     secure_environment_variables = {
-      "GIT_REMOTE"        = local.unipipe_git_remote
-      "GIT_REMOTE_BRANCH" = local.unipipe_git_branch
-      "GIT_SSH_KEY"             = local.unipipe_git_ssh_key
+      "GIT_REMOTE"              = local.unipipe_git_remote
+      "GIT_REMOTE_BRANCH"       = local.unipipe_git_branch
+      "GIT_SSH_KEY"             = tls_private_key.unipipe_git_ssh_key.private_key_pem
```

Comment:
does that work already? because IIRC there's still an open issue about reformatting line breaks in here
or does that work just fine when using an ENV var? 

Related issues: https://github.com/meshcloud/unipipe-service-broker/issues/33

If it works fine the way described here then we probably need to fix the description in https://github.com/meshcloud/unipipe-service-broker/blob/master/README.md#configuration which says "Linebreaks **must** be replaced with spaces."